uv.lock

```diff
@@ -1,5 +1,5 @@
 version = 1
-revision = 3
+revision = 2
```

Comment:
Looks like you are using an older version of uv, can update your copy with `uv pip install -U uv` or better yet sync with lockfile by doing `uv sync --extra tools --extra tests`, which will put all dependencies on exact versions whenever lockfile was last updated.

Reply:
That's alright, in the end it doesn't really matter, just noticed the revision number went down